src/angularComponent/ngcOmnibox.html

```diff
@@ -1 +1,5 @@
-<h1>Hello world!</h1>
+<input type="text"
+    ng-model="omnibox.ngModel"
+    ng-attr-placeholder="{{::omnibox.placeholder || undefined}}"
+    ng-attr-autofocus="{{omnibox.autofocus === 'true' || undefined}}"
```

Comment:
On the flip side, I think **this** should be bind-once, because changing the value later doesn't do anything -- it's already loaded.

Also, why normalize `false` to `undefined`? Isn't `false` just better?